src/utils/response.js

```diff
@@ -0,0 +1,54 @@
+const codes = {
+  200: 'sucess',
+  201: 'created',
+  400: 'bad_request',
+  401: 'unauthorized',
+  403: 'forbidden',
+  404: 'not_found',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
  404: "not_found",
```